package.json

```diff
@@ -17,23 +17,22 @@
     "dist/**"
   ],
   "scripts": {
-    "build": "yarn clean && ./bin/build",
+    "build": "npm run clean && ./bin/build",
     "clean": "rimraf dist",
-    "prepack": "yarn build",
-    "prepare": "yarn build",
+    "prepack": "npm run build",
```

Comment:
Do we need both `prepare` and `prepack` to do the same thing? Will this end up building the package twice upon publishing?

Reply:
I removed the `prepack` script